.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -124,7 +124,36 @@ stages:
       displayName: Generate Matrices
       name: generate_variables_step
 
-- stage: build_windows
+- stage: build_windows_tracer
+  dependsOn: []
+  jobs:
+  - template: steps/update-github-status-jobs.yml
+    parameters:
+      jobs: [build]
+
+  - job: build
+    pool:
+      vmImage: windows-2019
+    steps:
+    - checkout: self
+      path: s
+
+    - template: steps/install-latest-dotnet-sdk.yml
+    - script: tracer\build.cmd BuildTracerHome
+      displayName: Build tracer home
+
+    - script: tracer\build.cmd BuildMonitoringHome
+      displayName: Build monitoring home
```

Comment:
This requires _both_ the tracer and the profiler, will need to be in a subsequent task (possibly can be done in the package windows step?)

Reply:
After conversation, resolving this comment, since `BuildMonitoringHome` is not depends on tracer and profiler.
It just a bad name and description.

Reply:
> It just a bad name and description.

Should we fix that? 😅